source/02-layouts/breadcrumb/breadcrumb.stories.jsx

```diff
@@ -1,17 +1,22 @@
+import React from 'react'
+import ReactDOMServer from 'react-dom/server';
 import parse from 'html-react-parser';
 
 import twigTemplate from './breadcrumb.twig';
 import data from './breadcrumb.yml';
+import ContentPlaceholder from '../../01-global/content-placeholder/content-placeholder';
 
 const settings = {
   title: 'Layouts/Breadcrumb',
-  argTypes: {
-    is_demo: {
-      table: {
-        disable: true,
-      },
-    },
-  },
+  decorators: [
+    (Story, {args}) =>
+      <Story args={{
+        ...args,
+        breadcrumb_content: ReactDOMServer.renderToStaticMarkup(
+          <ContentPlaceholder>Breadcrumb Layout Content</ContentPlaceholder>
+        )
```

Comment:
This will forcibly override anything you put as `breadcrumb_content` in the args with placeholder content. What it seems like you're trying to do is provide `<ContentPlaceholder>` as the default arg for breadcrumb_content. In that case, I don't think a decorator is the right approach. I think instead you need to take `breadcrumb_content: ''` out of the YAML file and set Breadcrumb.args as 
```
Breadcrumb.args = {
  breadcrumb_content: ReactDOMServer.renderToStaticMarkup(
      <ContentPlaceholder>Breadcrumb Layout Content</ContentPlaceholder>
   ),
  ...data,
}
```
The ordering there is intentional so that if I do put content in breadcrumb_content in the YAML file, that content rather than the placeholder will be used.

You can still override the content anywhere you want to reuse the story by doing
```
Breadcrumb({
   ...Breadcrumb.args,
   breadcrumb_content: ReactDOMServer.renderToStaticMarkup(<div>Some different content here</div>)
})
```
Decorators are for wrapping a story in extra markup or functionality, like we were looking at with the buttons. 

This applies not just here but everywhere this pattern is used.

Reply:
That makes a lot of sense -- thank you! 

I've updated all the Layout and Template stories to match this updated pattern.  I left the component ones alone (back-to-top, button, icon, icon-link) because they all seemed more like decorators to me, but please let me know if you think additional changes are needed there or elsewhere.